Models/TodoContext.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using TodoApi.Models;
+
+    public class TodoContext : DbContext
+    {
+        public TodoContext (DbContextOptions<TodoContext> options)
+            : base(options)
+        {
+        }
+
+        public DbSet<TodoApi.Models.TodoItem> TodoItem { get; set; } = default!;
```

Comment:
TodoApi.Models provide this reference on top of the class as name space and then type should be defined

Reply:
Done